drivers/wallclock/wallclock.c

```diff
@@ -5,15 +5,17 @@
 
 #include <dm.h>
 #include <error.h>
+#include <stddef.h>
 #include <drivers/wallclock.h>
 
 static struct device *wallclock;
 
 int
 wallclock_device_register(struct device *dev)
 {
-	if (wallclock)
+	if (wallclock != NULL)
```

Comment:
Are we requiring `NULL` checks to be explicit? If so, we should add this to #28

Reply:
I don't think they should be required, no. Especially the idiom
```c
if ((err = foo()))
    handle_error(err);
```
is useful. I used to be against them ("redundant"), but other projects and programming languages have softened me up. So I'd say put them where you want them, but I won't complain either way.

Reply:
That is fine with me; I don't see any value in regulating it.